src/scikit_package/cli/gh.py

```diff
@@ -296,18 +330,37 @@ def _broadcast_issue_to_urls(issue_content, repo_urls, gh_token, dry_run=True):
     }
     non_gh_urls = []
     failed_gh_urls = []
+    success_gh_urls = []
     for i in range(len(repo_urls)):
         try:
             api_url = _get_post_api_url(repo_urls[i])
         except (IndexError, AssertionError):
             non_gh_urls.append(repo_urls[i])
+            print("Skipping non-GitHub URL:", repo_urls[i])
             continue
-        if not dry_run:
+        if dry_run:
+            success_gh_urls.append(repo_urls[i])
+        else:
             response = requests.post(api_url, json=data, headers=headers)
             if response.status_code != 201:
                 failed_gh_urls.append(repo_urls[i])
-    if dry_run:
-        _print_dry_run_message(non_gh_urls)
+                print(f"Failed to create issue in {repo_urls[i]}")
+                print(f"Response: {response.status_code}, {response.text}")
+            else:
+                success_gh_urls.append(repo_urls[i])
+    if len(success_gh_urls) > 0:
```

Comment:
Rather than printing the fails as they happen, I think it would be better to store them in failed_urls and jus summarize them here at the end after the successes, both in dry-run and otherweise.  For the dry-run case, is there a way we can do everything except actually make the issue, so any things that are likely to fail show up (presumably the most common will be typo's in the repo name)?